frontend/uddhar-frontend/src/coordinator/components/UserTable.jsx

```
@@ -1,73 +1,79 @@
+                        src={`https://i.pravatar.cc/40?img=${i + index}`}
+                        alt="Team Member"
+                      />
+                    ))}
+                  </td>
+                  <td className="p-3">
+                    <p className={`px-2 py-1 text-center ${statusClass}`}>
+                      {user.status}
+                    </p>
+                  </td>
+                  <td>
+                    <button className="border-2 rounded-[5px] bg-green-400 px-2 py-1 cursor-pointer text-black mr-1">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)